app/assets/stylesheets/govuk_publishing_components/components/_emergency-banner.scss

```diff
@@ -0,0 +1,74 @@
+.gem-c-emergency-banner {
+  @include govuk-font(19);
+  background-color: govuk-colour("mid-grey");
+  color: govuk-colour("white");
+  margin-top: 2px;
+  padding: govuk-spacing(3) 0;
+
+  @include govuk-media-query($from: tablet) {
+    padding: govuk-spacing(6) 0;
+  }
+
+  .homepage & {
+    border-bottom: govuk-spacing(1) solid govuk-colour("white");
+    border-top: govuk-spacing(1) solid govuk-colour("white");
+    margin-bottom: (0 - govuk-spacing(2));
+    margin-top: 0;
+    position: relative;
+    z-index: 10;
+  }
```

Comment:
Having the external `homepage` class affect the styles breaks the component principle of isolation[^1], specifically:

> it does not rely on external selectors to style its children

The only way around this I can think of doing this is by having this variation set by a parameter into the component, for example:

```erb
<%= render "govuk_publishing_components/components/emergency_banner", {
  campaign_class: "notable-death",
  heading: "His Royal Highness Henry VIII",
  short_description: "1491 to 1547",
  link_text: "Override Link Text",
  link: "https://www.gov.uk/",
  homepage: true,
} %>
```
 and then setting up a [homepage specific template in static](https://github.com/alphagov/static/tree/main/app/views/root) (`gem_layout_homepage_only.html.erb` for example) that only the homepage template in the `frontend` app uses. 

It's a bit more work now - but it keeps the isolation principle, and adding this functionality will be one less thing to need to update as Static is reduced in size.

[^1]: https://github.com/alphagov/govuk_publishing_components/blob/cec4f2f50dc0187e19d4e4a49c550ff0f7f6c4be/docs/component_principles.md#a-component-is-isolated-when